### PR TITLE
Restore the crunchy copy script to working order

### PR DESCRIPTION
**Pull request by tim-schilling**
This revises the readme to use a .env file approach for running terraform locally. This also uses a .env file in the project directory in the EC2 instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,39 +41,41 @@ export TF_VAR_ASPIRE_AWS_ACCESS_KEY_ID=YOUR_AWS_ACCESS_KEY
...
 
-5. (Optional but recommended) Verify the actions Terraform will take when applied with the following command:
+6. (Optional but recommended) Verify the actions Terraform will take when applied with the following command:
 
 ```
 terraform plan
 ```
 
-6. Run the following command to apply the changes from Step 5 and provision the specified resources from AWS:
+7. Run the following command to apply the changes from Step 5 and provision the specified resources from AWS:
```
Comment (Nish8192) [inline README.md:68]: "...from Step 5..." should now be "...from Step 6..." since the step numbers have been updated.

**Review by Nish8192 — APPROVED** [review]
One small update to the README but otherwise it looks good!

File: README.md (hunk 1 above)
Comment (tim-schilling) [reply README.md:68]: Good catch. By the way, in the files tab if you press `cmd+g`, you can suggest a code change that we can then commit from the github web UI.

Comment (tim-schilling) [inline README.md:68]:
```suggestion
7. Run the following command to apply the changes from Step 6 and provision the specified resources from AWS:
```

**Pull request merged**